.travis.yml

```diff
@@ -2,8 +2,20 @@ git:
   submodules: false
 language: node_js
 node_js:
-  - "8"
+- '8'
 install:
-  - npm install
-  - gulp views
-  - gulp styles
+- npm install
+- gulp views
```

Comment:
watcha thing about throwing `gulp views` and `gulp styles`  in the `script` block (i know thats not how it was before, but its the "correct" phase of the lifecycle) https://docs.travis-ci.com/user/job-lifecycle/

Reply:
I think "sure why not". Done.